go.mod

```diff
@@ -54,14 +55,14 @@ require (
 	github.com/qingstor/qingstor-sdk-go/v4 v4.4.0
 	github.com/qiniu/go-sdk/v7 v7.15.0
 	github.com/redis/go-redis/v9 v9.0.2
-	github.com/sirupsen/logrus v1.9.0
+	github.com/sirupsen/logrus v1.9.3
 	github.com/smartystreets/goconvey v1.7.2
 	github.com/stretchr/testify v1.8.4
 	github.com/studio-b12/gowebdav v0.0.0-20230203202212-3282f94193f2
 	github.com/tencentyun/cos-go-sdk-v5 v0.7.45
 	github.com/tikv/client-go/v2 v2.0.4
 	github.com/upyun/go-sdk/v3 v3.0.4
-	github.com/urfave/cli/v2 v2.19.3
+	github.com/urfave/cli/v2 v2.25.7
```

Comment:
Is this necessary? If not, we can upgrade it with another PR, so we can track the changes easier.

Reply:
Same to other dependencies.

Reply:
How would I do that? These changes are introduced when running `go get github.com/l0wl3vel/bunny-storage-go-sdk@v0.0.9`.

Reply:
Never mind if you did not upgrade them manually.